_inc/client/plans/single-product-backup.jsx

```diff
@@ -2,18 +2,63 @@
  * External dependencies
  */
 import React from 'react';
+import PropTypes from 'prop-types';
 import { translate as __ } from 'i18n-calypso';
 
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ExternalLink from 'components/external-link';
+
 import './single-product-backup.scss';
 
-export function PlanRadioButton( {
-	checked,
-	currencySymbol,
-	onChange,
-	planName,
-	radioValue,
-	planPrice,
-} ) {
+export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
```

Comment:
Seems like this component name is pretty generic but functionality-wise it's pretty backup-specific. Should we rename? Or should we allow for products to be specified and keep this as a simple presentational component?

Reply:
I'd rather make it specific, as I did not design this with it being generic in mind. We can always make it generic later if we need to.

Reply:
Sounds good to me 👍 Happy to change this name to a more specific one.